lego-webapp/pages/users/@username/_components/UserProfile.module.css

```
@@ -151,3 +155,25 @@ img.frame {
   height: 100px;
   width: auto;
 }
+
+.trophyContainer {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  gap: var(--spacing-sm);
+
+  @media (--lego-max-width) {
```

Biome (1.9.4)

[error] Don't use unknown media feature names.

Unexpected unknown media feature name.
You should use media feature names defined in the CSS Specifications.

(lint/correctness/noUnknownMediaFeatureName)